pkg/workload/connectionlatency/connectionlatency.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package connectionlatency
+
+import (
+	"context"
+
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+	"github.com/cockroachdb/cockroach/pkg/workload"
+	"github.com/cockroachdb/cockroach/pkg/workload/histogram"
+	"github.com/cockroachdb/errors"
+	"github.com/jackc/pgx"
+	"github.com/spf13/pflag"
+)
+
+type connectionLatency struct {
+	flags     workload.Flags
+	connFlags *workload.ConnFlags
+	hists     *histogram.Histograms
+}
+
+func init() {
+	workload.Register(connectionLatencyMeta)
+}
+
+var connectionLatencyMeta = workload.Meta{
+	Name:         `connectionlatency`,
+	Description:  `Testing Connection Latencies`,
+	Version:      `1.0.0`,
+	PublicFacing: false,
+	New: func() workload.Generator {
+		c := &connectionLatency{}
+		c.flags.FlagSet = pflag.NewFlagSet(`connectionlatency`, pflag.ContinueOnError)
+		c.connFlags = workload.NewConnFlags(&c.flags)
+		return c
+	},
+}
+
+// Meta implements the Generator interface.
+func (connectionLatency) Meta() workload.Meta { return connectionLatencyMeta }
+
+// Tables implements the Generator interface.
+func (connectionLatency) Tables() []workload.Table {
+	return nil
+}
+
+// Ops implements the Opser interface.
+func (c *connectionLatency) Ops(
+	ctx context.Context, urls []string, reg *histogram.Registry,
+) (workload.QueryLoad, error) {
+	ql := workload.QueryLoad{}
+	if len(urls) != 1 {
```

Comment:
is this always 1? it would be nice to be able to run this test in different configurations -- single-node, multi-node in 1 region, and multi-region. that way we can compare how it differs for all these.

take a look at some of the other tests on https://roachperf.crdb.dev/
some of them have `nodes=X` and `multiregion` specifiers. not sure how to get that type of configuration, but i think we want it here.